src/components/views/rooms/RoomListView/RoomListHeaderView.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+import React, { type JSX, useState } from "react";
+import { IconButton, Menu, MenuItem } from "@vector-im/compound-web";
+import ComposeIcon from "@vector-im/compound-design-tokens/assets/web/icons/compose";
+import UserAddIcon from "@vector-im/compound-design-tokens/assets/web/icons/user-add";
+import RoomIcon from "@vector-im/compound-design-tokens/assets/web/icons/room";
+import VideoCallIcon from "@vector-im/compound-design-tokens/assets/web/icons/video-call";
+
+import { _t } from "../../../../languageHandler";
+import { Flex } from "../../../utils/Flex";
+import {
+    type RoomListHeaderViewState,
+    useRoomListHeaderViewModel,
+} from "../../../viewmodels/roomlist/RoomListHeaderViewModel";
+
+/**
+ * The header view for the room list
+ * The space name is displayed and a compose menu is shown if the user can create rooms
+ */
+export function RoomListHeaderView(): JSX.Element {
+    const vm = useRoomListHeaderViewModel();
+
+    return (
+        <Flex
+            as="header"
+            className="mx_RoomListHeaderView"
+            aria-label={_t("room|context_menu|title")}
+            justify="space-between"
+            align="center"
+        >
+            <h1>{vm.title}</h1>
```

Comment:
I'm not sure about the `header` and `h1` tag. Is it better to have an `aside` instead?